core/base/src/TBuffer.cxx

```diff
@@ -35,12 +36,20 @@ static char *R__NoReAllocChar(char *, size_t, size_t)
 /// TBuffer::kWrite. By default the I/O buffer has a size of
 /// TBuffer::kInitialSize (1024) bytes.
 
-TBuffer::TBuffer(EMode mode)
+TBuffer::TBuffer(EMode mode, Bool_t def, Bool_t buffBigEndian)
```

Comment:
coding convention: the documentation (just before the function) must be extended explain the meaning of the new arguments.   When updating a function's documentation consider upgrading it to use the doxygen format.